src/DiyaNode.js

```diff
@@ -584,6 +609,13 @@ DiyaNode.prototype._handleSubscription = function(handler, message){
 	this._notifyListener(handler, message.error, message.data ? message.data : null);
 };
 
+DiyaNode.prototype._handleServerData = function(message){
+	try {
+		this._streamSocket[message.data.socketId].push(new Buffer(message.data.buffer, 'base64'));
```

Comment:
I think you should use ArrayBuffer (native HTML buffers) instead of NodeJS buffer if possible, as this will prevent pulling a huge dependency from the nodejs library...